README.md

```diff
@@ -29,28 +35,144 @@ $ git clone git@github.com:openfun/xblock-configurable-lti-consumer.git
 ```
 
 Once the project has been cloned on your machine, you will need to build a
-custom edx-platform docker image that includes the xblock and setup a
-development environment that includes all required services up and running (more
-on this later):
+custom [edx-platform docker image](https://github.com/openfun/openedx-docker) that
+includes the configurable LTI consumer XBlock and setup a development environment that
+includes all required services up and running (more on this later):
 
 ```bash
 $ cd xblock-configurable-lti-consumer
 $ make bootstrap
 ```
 
-If everything went well, you should now be able to access to the following
-services:
+If everything went well, you should now be able to access the following services:
 
 - Open edX LMS: http://localhost:8072
 - Open edX CMS: http://localhost:8082
 
 with the following credentials:
 
 ```
-email: admin@foex.edu
-password: openedx-rox
+email: admin@example.com
+password: admin
+```
+
+## Configuration examples
+
+The following LTI configuration does several things:
+
+- it adds a "Marsha Video" link behind the `Advanced` component button in the Studio to add a
+  video in one click. The video is automatically added to
+  [Marsha](https://github.com/openfun/marsha),
+- it forces all LTI consumer XBlocks that are pointing to `ltiapps.net` to use a modal, together
+  with a few other settings,
+- it forces all other LTI consumer XBlocks to open in a new window and ask before sending the
+  user's username and email.
+
+```yaml
+LTI_XBLOCK_CONFIGURATIONS:
+  # A configuration that adds a Marsha Video XBlock
+  - boilerplate: marsha
+    display_name: Marsha Video
+    oauth_consumer_key: InsecureOauthConsumerKey
+    shared_secret: InsecureSharedSecret
+    fields:
+      lti_id: marsha
+      description: ""
```

Comment:
Does it worth it to add a short description here?

Reply:
yes to hide this useless field!

Reply:
We have finally decided to split the list of hidden fields from the list of values for a field...